SUPPORT.md

```diff
@@ -0,0 +1,20 @@
+# Support Policy
+
+Version 1 of the Terraform Plugin SDK is considered **deprecated**, which means
+the following support policy is in place:
+
+- Critical bug fixes will be accepted and merged until July 31, 2021, at which
+point this version is considered End Of Life and no longer supported. New
+features and non-critical bug fixes will not be accepted.
+- We will not break the public interface exposed by the SDK in a minor or patch
+release unless a critical security issue demands it, and only then as a last
+resort.
+- We do not proactively test version 1 of the SDK against new releases of
+- Terraform to ensure compatibility, but will address critical compatibility
```

Comment:
Seems to be a stray line break + bullet here

Reply:
Thanks. Fixed!